superset-frontend/src/dashboard/styles.ts

```diff
@@ -97,3 +111,43 @@ export const chartContextMenuStyles = (theme: SupersetTheme) => css`
     min-width: ${theme.gridUnit * 40}px;
   }
 `;
+
+export const focusStyle = (theme: SupersetTheme) => css`
+  a,
+  .ant-tabs-tabpane,
+  .ant-tabs-tab-btn,
+  .superset-button.action-button,
+  .superset-button.ant-dropdown-trigger,
+  .header-controls span {
+    &:focus-visible {
+      box-shadow: 0 0 0 2px ${theme.colors.primary.dark1};
+      border-radius: ${theme.gridUnit / 2}px;
+      outline: none;
+      text-decoration: none;
+    }
+    &:not(.superset-button, .ant-menu-item
+        a, .fave-unfave-icon, .ant-tabs-tabpane, .header-controls span) {
+      &:focus-visible {
+        padding: ${theme.gridUnit / 2}px;
+      }
+    }
+  }
+
+  @keyframes fadeOut {
```

Comment:
Oooooh, fancy! Just another "note to self" that we can add shadows to the theme as well, so they can be appropriate "glows" in dark mode, and be standardized across the UI.

Reply:
If you wanna have some fun, you could still add a base box-shadow style to the component with no visible shadow, and add a `transition` property so that the shadow fades on and/or moves when the `focus-visible` pseudoclasss is applied.

Reply:
I think we're ok with the outline when someone tabs into an element, but it just didn't feel right when clicking on an element, especially when the click bubbles up to an outer element and you have no idea why the entire tab is highlighting.